src/screens/About/About.tsx

```diff
@@ -103,6 +107,11 @@ const About = () => {
           <MarkdownContent source={aboutContent.teamIntro} />
           <TeamSection />
         </HashWrapper>
+        <HashWrapper id="future-projects">
+          <AboutHeading2>{aboutContent.futureProjectsHeader}</AboutHeading2>
+          <MarkdownContent source={aboutContent.futureProjectsContent} />
+          <TeamSection />
```

Comment:
The Team section is duplicated here, everyone shows up twice in the page

Reply:
oh, yikes.  Good catch.